x/gov/legacy/v4/store.go

```diff
@@ -0,0 +1,51 @@
+package v4
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+// If expedited, the deposit to enter voting period will be
+// increased to 5000 OSMO. The proposal will have 24 hours to achieve
+// a two-thirds majority of all staked OSMO voting power voting YES.
+var minInitialDepositRatio = sdk.NewDec(25).Quo(sdk.NewDec(100))
+var minExpeditedDeposit = sdk.NewCoins(sdk.NewCoin("osmo", sdk.NewInt(5000)))
```

Comment:
Hmm this is not ideal that we have this coupling in the sdk on osmosis by having to hardcode the osmo denom here.

This is a flaw in the original design though and regular deposits have the same problem. I think it's okay to hard code it here, just commenting to keep this issue in mind